examples/src/main/kotlin/usageExamples/update/Update.kt

```diff
@@ -0,0 +1,55 @@
+package usageExamples.update
+
+// :replace-start: {
+//    "terms": {
+//       "CONNECTION_URI_PLACEHOLDER": "\"<connection string uri>\"",
+//       "import io.github.cdimascio.dotenv.dotenv\n": ""
+//    }
+// }
+// :snippet-start: update-usage-example
+
+import com.mongodb.MongoException
+import com.mongodb.client.model.Filters
+import com.mongodb.client.model.UpdateOptions
+import com.mongodb.client.model.Updates
+import com.mongodb.kotlin.client.coroutine.MongoClient
+import io.github.cdimascio.dotenv.dotenv
+import kotlinx.coroutines.runBlocking
+import java.util.*
+
+data class Movie(val title: String, val runtime: Int, val genres: List<String>, val lastUpdated: Date)
+
+fun main() = runBlocking {
+    // :remove-start:
+    val dotenv = dotenv()
+    val CONNECTION_URI_PLACEHOLDER = dotenv["MONGODB_CONNECTION_URI"]
+    // :remove-end:
+    // Replace the uri string with your MongoDB deployment's connection string
+    val uri = CONNECTION_URI_PLACEHOLDER
+    val mongoClient = MongoClient.create(uri)
+    val database = mongoClient.getDatabase("sample_mflix")
+    val collection = database.getCollection<Movie>("movies")
+
+    collection.insertOne(Movie("Cool Runnings 2", 90, listOf("Adventure", "Family", "Comedy"), Date())) // :remove:
+    val query = Filters.eq(Movie::title.name, "Cool Runnings 2")
+    val updates = Updates.combine(
+        Updates.set(Movie::runtime.name, 99),
+        Updates.addToSet(Movie::genres.name, "Sports"),
+        Updates.currentTimestamp(Movie::lastUpdated.name)
```

Comment:
```suggestion
        Updates.currentDate(Movie::lastUpdated.name)
```

Reply:
Updated in both Update.kt and UpdateMany.kt